ibm/resource_ibm_cis_rate_limit.go

```diff
@@ -145,14 +145,14 @@ func resourceIBMCISRateLimit() *schema.Resource {
 			},
 			"match": {
 				Type:        schema.TypeList,
-				Optional:    true,
+				Required:    true,
 				MaxItems:    1,
 				Description: "Rate Limiting Match",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"request": {
 							Type:        schema.TypeList,
-							Optional:    true,
```

Comment:
revert back the changes
please donot make changes in existing schema

Reply:
done

Reply:
But, "request" property is a mandatory field to rate limit. if user did not specify request property in tf code, they will get a crash log. 

I made it optional, since, you have asked to make it optional.

Reply:
made the request field as optional and computed.
modified the expandMatch code to set the default match request value.
Now it works if we don't specify match and request property.